fvtest/compilertriltest/ArithmeticTest.cpp

```diff
@@ -808,6 +816,235 @@ INSTANTIATE_TEST_CASE_P(ArithmeticTest, FloatArithmetic, ::testing::Combine(
         std::make_tuple<const char*, float (*)(float, float)>("fdiv", static_cast<float (*)(float, float)>(div))
     )));
 
+#ifdef OMR_ENV_DATA64
+/**
+ * uint64_t is used as the first argument type here, because aiadd will only 
+ * be used on 64 bit platforms(checked by verifier). Address will be a uint64_t type in this case. 
```

Comment:
Typo
```suggestion
 * uint64_t is used as the first argument type here, because aladd will only 
 * be used on 64 bit platforms(checked by verifier). Address will be a uint64_t type in this case. 
```